content/en/docs/tasks/manage-gpus/scheduling-gpus.md

```diff
@@ -64,29 +64,40 @@ spec:
           gpu-vendor.example/example-gpu: 1 # requesting 1 GPU
 ```
 
-## Clusters containing different types of GPUs
+## Automatically labeling nodes with Node Feature Discovery {#node-feature-discovery}
 
-If different nodes in your cluster have different types of GPUs, then you
-can use [Node Labels and Node Selectors](/docs/tasks/configure-pod-container/assign-pods-nodes/)
-to schedule pods to appropriate nodes.
+As and administrator, you can automatically discover and label all your GPU enabled nodes
+by deploying the K8S-Sig project Node Feature Discovery [NFD](https://github.com/kubernetes-sigs/node-feature-discovery).
+NFD enables node feature discovery for Kubernetes. 
+It detects hardware features available on each node in a Kubernetes cluster, and advertises those features using node labels and optionally node extended resources, annotations and node taints. 
+Node Feature Discovery is compatible with any recent version of Kubernetes (v1.21+).
```

Comment:
I'll rarely critique using a full project name, but here we've already abbreviated, so I start with "NFD is..."  However, I'd wait to hear from a maintainer about whether and how to mention version information.  The [Versioning Kubernetes examples](https://kubernetes.io/docs/contribute/style/style-guide/#versioning-kubernetes-examples) topic doesn't apply, I don't think.  v1.21 is already EOL, IIUC.